tests/projects.ts

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
I refactored some "test data setup code" from `tests/docker-build.ts` to `tests/projects.ts` because I felt the code was more specific to the `tests/test-data/projects/` test data than specific to "docker build".